test/test_datasets_vctk.py

```diff
@@ -23,10 +23,13 @@ def test_is_audio_file(self):
     def test_make_manifest(self):
         audios = vctk.make_manifest(self.test_dirpath)
         files = ['kaldi_file.wav', 'kaldi_file_8000.wav',
-                 'sinewave.wav', 'steam-train-whistle-daniel_simon.mp3']
+                 'sinewave.wav', 'steam-train-whistle-daniel_simon.mp3',
+                 'dtmf_30s_stereo.mp3', 'whitenoise_1min.mp3', 'whitenoise.mp3']
```

Comment:
Why is this change here?

Reply:
We added more audio files (generated files, license free) for testing.  The VCTK test was failing.  Is sinewav, steam-train-whistle, actually a dataset or its just checking which files it sees in the directory.

Reply:
I see. This test is only to see that the manifest is generated correctly with the given test folder. Indeed, we added files, and did not update the test.